it/test/uk/gov/hmrc/timetopayproxy/config/DocumentationSpec.scala

```diff
@@ -26,13 +26,6 @@ class DocumentationControllerSpec extends IntegrationBaseSpec {
   val apiDefinitionJson: JsValue = Json.parse(
     """
       |{
-      |  "scopes": [
-      |    {
-      |      "key": "read:time-to-pay-proxy",
-      |      "name": "Time to pay proxy",
-      |      "description": "Allow read access to time to pay proxy"
-      |    }
-      |  ],
```

Comment:
👍 

Also checked this against [the common JSON file](https://github.com/hmrc/api-scope/blob/5b55e92029bcede93d5acf1852696c5c8d3d91e5/conf/scopes.json#L1015).

![image](https://github.com/user-attachments/assets/7f7c5dbf-a4dd-44b3-8e1f-d60d1f8a73f2)
(from [the announcement in Confluence](https://confluence.tools.tax.service.gov.uk/display/TEC/2021/09/07/Changes+to+scopes))